specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/managedCassandra.json

```diff
@@ -341,6 +341,139 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/invokeCommandAsync": {
+      "post": {
+        "operationId": "CassandraClusters_InvokeCommandAsync",
+        "description": "Invoke a command like nodetool for cassandra maintenance asynchronously",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraCommandAsync": {
+            "$ref": "./examples/CosmosDBManagedCassandraCommandAsync.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommandPostBody"
+            },
+            "description": "Specification which command to run where"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success. The command operation will begin as soon as possible.",
+            "schema": {
```

Comment:
202 response for a LRO POST operation must not have a response schema specified.
https://github.com/Azure/azure-openapi-validator/blob/main/docs/post-response-codes.md